backend/v3/domain/org_list_test.go

```diff
@@ -425,7 +425,7 @@ func TestListOrgsCommand_ResultToGRPC(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			t.Parallel()
 
-			cmd := &domain.ListOrgsCommand{
+			cmd := &domain.ListOrgsQuery{
 				Result: tc.orgs,
```

Comment:
Needs to be re-exported

Reply:
we will get rid of this test anyways when [this comment](https://github.com/zitadel/zitadel/pull/10704#discussion_r2416485620) gets resolved.